python/lsst/ts/standardscripts/maintel/take_image_comcam.py

```diff
@@ -48,8 +49,13 @@ def __init__(self, index):
 
         self.config = None
 
+        self.mtcs = MTCS(self.domain, log=self.log)
```

Comment:
Same comment as above for the auxtel script, but it is a lot more critical for MT, you need to limit the usages or it will use way too many resources. It should be something like:

```
self.mtcs = MTCS(self.domain, log=self.log, intended_usage=MTCSUsages.Slew)
```